shared/fs/folder/index.js

```diff
@@ -8,7 +8,7 @@ import FolderHeader from '../header/container'
 import Footer from '../footer/footer'
 import {isMobile} from '../../constants/platform'
 import Rows from '../row/rows-container'
-import DropTarget from './drop-target'
+import DragAndDrop from '../../common-adapters/drag-and-drop.desktop'
```

Comment:
you should make common-adapters/electron.desktop.js (like mobile.native.js) so we can put desktop specific stuff in there

Reply:
KBFS is using this in a place shared between desktop and mobile. So, I copied the `SafeAreaView` pattern instead. Let me know if you'd prefer me to do something else, though.

Reply:
if its native or shared it can't pull in .desktop files, is that what you mean?

Reply:
Right. So, I'm only rendering the children on native: https://github.com/keybase/client/pull/16535/files#diff-dd41b322f4d13e08cb35827f54986e11R4